assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_new_edit_assignment.vm

```diff
@@ -26,7 +26,9 @@
 		 		});
 		 	});
 		#end
-		SetMaxPoint(document.getElementById('$name_GradeType'), document.getElementById('$name_GradePoints'));
+		if ($("#generalAlert").length <= 0) {
```

Comment:
$PBJQ work instead of $?

Reply:
Is one better than the other? There are no occurrences of 'PBJQ' in assignments currently.

Reply:
I don't fully understand it but @csev has commented on why we use PBJQ in the past

Reply:
Well anything that uses the portal's jquery (and is in library) needs to use PBJQ to make sure to use the correct jquery since they're compiled into morpheus scripts. Some tools (like Lessons) are still on an older local version of JQuery which is what the $ is. You can look at the javascript console in a tool to *see* what it's using. Like for instance 

Lessons:
Portal script load complete PBJQ=1.12.4 jQuery=1.9.1 $=1.9.1

Assignment:
Portal script load complete PBJQ=1.12.4 jQuery=1.12.4 $=1.12.4

So for most it doesn't matter $ === jQuery === PBJQ, but some tools it doesn't and when you do things like extend functionality you want to make sure you're on the right jQuery.

Reply:
What we really need to do is clean up the tools that use these old versions of JQ, Lessons should be easy. Evaluation maybe harder. So in any case not using PBJQ here since it's not currently used is fine.

Reply:
That makes a little more sense, thanks for the explanation @jonespm. So considering there are currently no existing occurances of 'PBJQ' in assignments code, is continuing to use $ a show stopper here?

Reply:
Nope, fine here since it's just local to assignment and just needs whatever jquery version assignment has.

Reply:
Thanks, gents!